### DISCUSSION TITLE
docker compose: Receipes unreachable after short time

### DISCUSSION THREAD

**Discussion by BstAA** [TandoorRecipes/recipes#3394] [Q&A]
Situation: recipes running under docker compose as in official docs. Problems:

- after some time (~day) recipes is unreachable. All containers running. Nothing to see in the logs, To repair only `docker compose down / docker compose up -d` helps
- after running `docker compose pull` the standard way is to just use `docker compose up -d`. Recipes will only start when using `docker compose down`

**Comment by smilerz** [comment]
You would be better served to seek support on a docker related channel.  This isn't related to Tandoor.
